docs/reference-guides/best-practices/rancher-managed-clusters/tips-to-set-up-containers.md

```diff
@@ -50,7 +54,10 @@ If you set a resource limit for the namespace that the pod is deployed in, and t
 
 It is recommended to define resource requirements on the container level because otherwise, the scheduler makes assumptions that will likely not be helpful to your application when the cluster experiences load.
 
+TODO we recommend to set your CPU request at 50m and memory request at 100Mi, however this is a base recommendation and customers should choose based on resources available in their environment and their use case on how they use Rancher.
```

Comment:
I don't think I phrased this particularly well, how about:

_As a base line we recommend setting the CPU request at 50m and memory request at 100Mi. However, it is important that you ensure your cluster has the correct resources available and the use case has been assessed appropriately before doing so._